### DISCUSSION TITLE
Local postgres on Macbook

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#90] [General]
Hi, 
Have u guys tried running this project on Macbook with local postgres.

I am trying now, it is showing Connection refused. I don't know if its the issue with my postgres or the project.

im having a fresh Mac with docker and postgres installed. 
do i need to install any other things, for this to work ?

anyone has successfully run using local postgres on Mac ?

**Comment by vincue** [comment]
Hi @jithendra945, the problem is as described [here](https://forums.docker.com/t/net-host-does-not-work/17378). We are looking for a workaround and will get back to you. Meanwhile you can continue with sqlite and feel free to contrubute if you find a solution.

**Reply by vincue** [reply]
Hi @jithendra945, I guess you're facing issues with data persistance on using default sqlite db. For now you can create a sqlite3 db named `db.sqlite3` and use this command for docker run 
`docker run -v /<filepath of db.sqlite3>:/code/db.sqlite3 -dp 3000:3000 cuebook/cueobserve`

**Comment by jithendra945** [comment]
POSTGRES_DB_SCHEMA=postgres
POSTGRES_DB_USERNAME=postgres
POSTGRES_DB_PASSWORD=postgres
POSTGRES_DB_HOST=docker.for.mac.host.internal
POSTGRES_DB_PORT=5432


docker run --env-file .env -p 5432:5432 -dp 3000:3000 cuebook/cueobserve


This is working to connect to local postgres on Mac.
Please Check it out once.

**Reply by vincue** [reply]
Don't have a mac at access, it won't work probably & must throw some error like `bind: address already in use`.

**Reply by jithendra945** [reply]
Nope. Im running with it.
It is working fine as with sqlite.

Will post again, if found any issue
